src/internal_rep.rs

```diff
@@ -500,16 +618,52 @@ impl TypeInfo {
     // specifically associations specifically in source, rather than somehow derived by the
     // compiler
     pub fn explicitly_associates(&self, associate_name: &str) -> Option<Range<usize>> {
+        use crate::compile::get_synthetic_resource_name;
+
         for ann in &self.annotations {
-            if let AnnotationInfo::Associate(associations) = ann {
+            if let AnnotationInfo::Associate(associations)
+            | AnnotationInfo::NestAssociate(associations) = ann
+            {
                 for res in &associations.resources {
-                    if res.as_ref() == associate_name && res.get_range().is_some() {
-                        return res.get_range();
+                    if res.string_is_instance(&CascadeString::from(associate_name))
+                        && res.get_range(associate_name).is_some()
+                    {
+                        // I think the issue is that this get_range() is returning None
+                        // In other news, I'm an idiot.  We are in a block for an if .is_some() on
```

Comment:
Is this still the case with the `get_range(associate_name)` call?

Reply:
Oh, I definitely intended to remove the "I'm an idiot" comment :)

This (the comment) was all just wrong from the get-go, as explained in the comment.  I'll delete.